### ISSUE TITLE
DockerContainersEnumerationTask fails in cloud e2e test

### ISSUE THREAD

**Issue by hacktobeer** [google/turbinia#803]
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/psq/task.py", line 90, in execute
    result = self.call_func()
  File "/usr/local/lib/python3.6/dist-packages/psq/task.py", line 104, in call_func
    return self.f(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210331.dev2+ga3a2d6c-py3.6.egg/turbinia/task_manager.py", line 97, in task_runner
    return obj.run_wrapper(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210331.dev2+ga3a2d6c-py3.6.egg/turbinia/workers/__init__.py", line 867, in run_wrapper
    with task_runtime_metrics.time():
AttributeError: 'NoneType' object has no attribute 'time'
2021-04-14 09:13:53 [ERROR] Task fcf8f8195165441ea15fea21b86601d1 failed.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/psq/task.py", line 90, in execute
    result = self.call_func()
  File "/usr/local/lib/python3.6/dist-packages/psq/task.py", line 104, in call_func
    return self.f(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210331.dev2+ga3a2d6c-py3.6.egg/turbinia/task_manager.py", line 97, in task_runner
    return obj.run_wrapper(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/turbinia-20210331.dev2+ga3a2d6c-py3.6.egg/turbinia/workers/__init__.py", line 867, in run_wrapper
    with task_runtime_metrics.time():
AttributeError: 'NoneType' object has no attribute 'time'
```

The complete job hangs as the DockerContainersEnumerationTask never returns/writes any status so the server thinks it's still running (while it's not and already failed on the worker).

**Comment by hacktobeer** [comment]
It's still unclear to me what the above error has to do with the DockerEnumeration task. Other tasks execute this code as well to initialise the task metrics, there is nothing different. 
Could it be a 'race condition' where this is the first task initialised before the global metrics var has been initialised?

**Comment by hacktobeer** [comment]
[worker0.log](https://github.com/google/turbinia/files/6310411/worker0.log)
[server.log](https://github.com/google/turbinia/files/6310412/server.log)
[worker1.log](https://github.com/google/turbinia/files/6310413/worker1.log)

**Comment by hacktobeer** [comment]
Fixed in #808
